### ISSUE TITLE
Improve default error message?

### ISSUE THREAD

**Issue by igoradamenko** [postcss/postcss#1057]
Hi,

If plugin throws an error that PostCSS can't recognize, it checks versions of plugin & core and throws default message, that looks so:

```
Your current PostCSS version is <x>, but <plugin> uses <y>. Perhaps this is the source of the error below.
```

I see that it contains `perhaps`, but it's not clear enough. Is it possible to change default message like this (or more precisely)?

```
Unknown error. Your current PostCSS version is <x>, but <plugin> uses <y>. Perhaps this is the source of the error below.
```

'cause right now it does not look like 'Hey man, we don't know what it is, but check versions just in case.', it looks like 'Something does wrong and most likely because of different versions.'

In my case, I made a mistake in autoprefixer config (passed `[['>1%']]` as value of `browsers` option), and get error message as above, 'cause I have different autoprefixer & postcss versions (for reasons beyond my control, but they worked fine). As you can guess, I reinstalled all project deps, changed version of postcss / autoprefixer / whole university, but nothing was fixed 😞 

So I want to suggest to improve error message to make it more clearly.

**Comment by ai** [comment]
Yeap, let’s improve it. You want just add `Unknown error`?

**Comment by igoradamenko** [comment]
Yes, if you don't have a better solution.

**Comment by ai** [comment]
Maybe `Unknown error from PostCSS plugin`?

**Comment by igoradamenko** [comment]
Yeah, it looks better!

**Comment by ai** [comment]
@igoradamenko OK, send PR

**Comment by igoradamenko** [comment]
Done: #1058.

**Comment by ai** [comment]
Thanks. Released in 6.0.9.
